src/Webapi/Dom/Webapi__Dom__DataTransfer.res

```diff
@@ -0,0 +1,35 @@
+type t = Dom.dataTransfer
+
+type dropEffect = [#none | #copy | #link | #move]
+
+type effectAllowed = [
```

Comment:
I wonder how much type safety we want on these things. It could be generic strings and not have them as polymorphic variants too. I think a lot of the webapi is like that already not sure if that is good or bad. It's more future proof if things gets added but less type safe.

Reply:
Yeah I had a similar point in https://github.com/tinymce/rescript-webapi/discussions/23, maybe a `#future_added_value` could be useful like how `graphql-ppx` encodes enums?

Reply:
I don't see this list changing anytime soon, it has been this way since 2008.

Having a catch-all future value doesn't make sense with polymorphic variants - they're compiled to strings. This is a case where I think zero cost is the correct approach.

Reply:
What I mean it would be a plain string. But I guess why some of these things are just strings in the current bindings are more that the poly variants didn't map to strings when it was written. So I guess we should aim for strong types.